pyodide_lock/spec.py

```diff
@@ -61,3 +61,29 @@ def to_json(self, path: Path, indent: int = 0) -> None:
         """Write the lock spec to a json file."""
         with path.open("w") as fh:
             json.dump(self.dict(), fh, indent=indent)
+
+    def check_wheel_filenames(self) -> dict[str, list[str]]:
```

Comment:
quick question: Why return errors in dictionary instead of throwing an error?

Reply:
Thanks for the review! Yeah, you are right it would be simpler to just raise an error here. I guess I was still hesitant as to whether this check should run on the top level or on the package level hence this version. But I'll switch to raising an error.